gradle-config.gradle

```diff
@@ -0,0 +1,22 @@
+//
+// Configuration file for gradle build execution.
+//
+
+task accessToken {
+  def tokenFile = new File("${projectDir}/src/main/res/values/developer-config.xml")
+  if (!tokenFile.exists()) {
+    String tokenFileContents = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n" +
+      "<resources>\n" +
+      "    <string name=\"mapbox_access_token\">" + "$System.env.MAPBOX_ACCESS_TOKEN" + "</string>\n" +
+      "</resources>"
+
+    if (tokenFileContents == null) {
```

Comment:
@cammace @Guardiola31337 This is never true, right? Even if `$System.env.MAPBOX_ACCESS_TOKEN` isn't set, `tokenFileContents` will have some content.